### ISSUE TITLE
Code of conduct: update email @dartlang.org address to @dart.dev

### ISSUE THREAD

**Issue by chalin** [dart-lang/site-www#1935]
Page URL: https://dart.dev/community/code-of-conduct
Page source: https://github.com/dart-lang/site-www/tree/master/src/code-of-conduct.md

---

Use `conduct@dart.dev` in the following `mailto` link?

    ... please [email us](mailto:conduct@dartlang.org).

cc @kwalrath

**Comment by kwalrath** [comment]
I believe the address is still `@dartlang.org`.

/cc @timsneath @messi01

**Comment by micpap25** [comment]
Is this still an issue? The email hasn't changed from `conduct@dartlang.org`. This could be closed.

**Comment by chalin** [comment]
You're right that conduct@dartlang.org is still active (I just tested it). It's more a question of whether it _should_ be updated to refer to dart.dev, which might happen eventually (I can understand that it's low priority). Is there an urgency to close this issue? Cheers

**Comment by micpap25** [comment]
It seems like a very easy fix, I could do it. But I agree, I'll defer to someone who maintains the repo regarding whether this should be updated.

**Comment by kwalrath** [comment]
I expect @mit-mit to update us soon on the feasibility of changing the addresses from dartlang.org to dart.dev.

**Comment by MrBirb** [comment]
@mit-mit Any updates on this Michael?

**Comment by atsansone** [comment]
@mit-mit : This email is still incorrect on this page. Would you like it fixed?

**Comment by mit-mit** [comment]
I've not been able to get the updated email account for this to work, so I think we just have to live with this...

**Comment by mit-mit** [comment]
Re-activating for a second attempt
